alm/src/config/constants.ts

```diff
@@ -0,0 +1,22 @@
+export const HOME_BRIDGE_ADDRESS: string = process.env.REACT_APP_COMMON_HOME_BRIDGE_ADDRESS || ''
+export const FOREIGN_BRIDGE_ADDRESS: string = process.env.REACT_APP_COMMON_FOREIGN_BRIDGE_ADDRESS || ''
+
+export const HOME_RPC_URL: string = process.env.REACT_APP_COMMON_HOME_RPC_URL || ''
```

Comment:
most probably you will need configuration parameters to build links to Etherscan and Blockscout. We have similar ones in the UI's `.env` file.

Reply:
Added variables for tx links template https://github.com/poanetwork/tokenbridge/pull/353/commits/475cf8a338d1d03b2266b100e57fa6a930c78e80